src/pages/Profile.js

```diff
@@ -355,22 +356,131 @@ const PassportLinker = ({ queryParams: { state, code } = {}, provider, prettyNam
+    div({ style: { flex: 1 } }, [
+      div({ style: { fontWeight: 600 } }, 'Name')
+    ]),
+    div({ style: { flex: 1 } }, [
+      div({ style: { fontWeight: 600 } }, 'Opt In')
+    ])
+  ])
+})
+
+const NotificationCard = memoWithName('NotificationCard', ({ label, notificationKeys, setSaving, prefsData }) => {
+  const notificationCardStyles = {
+    field: {
+      ...Style.noWrapEllipsis, flex: 1, height: '1rem', paddingRight: '1rem'
+    },
+    row: { display: 'flex', alignItems: 'center', width: '100%', padding: '1rem' }
+  }
+
+  return div({ role: 'listitem', style: { ...Style.cardList.longCardShadowless, padding: 0, flexDirection: 'column' } }, [
+    div({ style: notificationCardStyles.row }, [
+      div({ style: { ...notificationCardStyles.field, display: 'flex', alignItems: 'center' } }, label),
+      div({ style: notificationCardStyles.field }, [h(NotificationCheckbox, { notificationKeys, setSaving, prefsData })])
+    ])
+  ])
+})
+
+const NotificationSettingsTab = ({ setSaving }) => {
+  const { workspaces } = useWorkspaces()
+  const [prefsData] = _.over(_.pickBy)((_v, k) => _.startsWith('notifications/', k), authStore.get().profile)
+
+
+  return h(PageBox, { role: 'main', style: { flexGrow: 1 }, variant: PageBoxVariants.LIGHT }, [
+    div({ style: Style.cardList.toolbarContainer }, [
+      h2({ style: { ...Style.elements.sectionHeader, margin: 0, textTransform: 'uppercase' } }, [
+        'Account Notifications',
+        h(InfoBox, { style: { marginLeft: '0.5rem' } },
+          'You may receive email notifications regarding certain events in Terra. You may globally opt in or out of receiving these emails.'
+        )
+      ])
+    ]),
+    h(NotificationCardHeaders),
+    div({ role: 'list', 'aria-label': 'notification settings for your account', style: { flexGrow: 1, width: '100%' } }, [
+      h(NotificationCard, {
+        setSaving,
+        prefsData,
+        label: 'Group Access Requested',
+        notificationKeys: ['notifications/GroupAccessRequestNotification']
+      }),
+      h(NotificationCard, {
+        setSaving,
+        prefsData,
+        label: 'Workspace Access Added',
+        notificationKeys: ['notifications/WorkspaceAddedNotification']
+      }),
+      h(NotificationCard, {
+        setSaving,
+        prefsData,
+        label: 'Workspace Access Removed',
+        notificationKeys: ['notifications/WorkspaceRemovedNotification']
+      })
+    ]),
+    div({ style: Style.cardList.toolbarContainer }, [
+      h2({ style: { ...Style.elements.sectionHeader, marginTop: '2rem', textTransform: 'uppercase' } }, [
+        'Submission Notifications',
+        h(InfoBox, { style: { marginLeft: '0.5rem' } },
+          'You may receive email notifications regarding terminal submissions in Terra (i.e. if a submission has succeeded, failed, or been aborted). You may opt in or out of receiving these emails for individual workspaces.'
```

Comment:
"terminal submissions" seems kind of jargony.

```suggestion
          'You may receive email notifications when a submission has succeeded, failed, or been aborted. You may opt in or out of receiving these emails for individual workspaces.'
```